src/biogeophys/CanopyFluxesMod.F90

```diff
@@ -1624,7 +1624,8 @@ subroutine CanopyFluxes(bounds,  num_exposedvegp, filter_exposedvegp,
       ! snocan < rel_epsilon * snocan_baseline will be set to zero
       ! See NumericsMod for rel_epsilon value
       call truncate_small_values(fn, filterp, begp, endp, &
-         snocan_baseline(begp:endp), snocan(begp:endp))
+         snocan_baseline(begp:endp), snocan(begp:endp), &
+         custom_rel_epsilon=1.e-10_r8)
```

Comment:
The default value is 1e-13. A value of 1e-12 or 1e-11 would probably be sufficient here... maybe it wouldn't truncate the state to 0 in the first possible time step, but it probably would be triggered before the tiny value became an issue... but I felt like it would be safer to add a few orders of magnitude of slop. The risk of making this too big is that we could truncate values to zero that really are supposed to be non-zero, but even with 1e-10 we're still only triggering the truncation if the post-state-update value is 10 orders of magnitude smaller than the initial value, so this still seems *pretty* safe to me.

Reply:
We'll discuss this as part of #2458. Bringing this in like this makes sense to me for now.

I'm resolving this to free this PR up...